test/benchmark-udp-pummel.c

```diff
@@ -145,6 +145,14 @@ static void timeout_cb(uv_timer_t* timer) {
 }
 
 
+char* build_expected_buf(int number) {
+  char* str = NULL;
+  strcpy(str, EXPECTED);
+  strcpy(str, (char*)&number);
```

Comment:
This isn't equivalent to `EXPECTED + number` - in fact, it's UB.

`&EXPECTED[number]` is probably what you're looking for to silence the compiler warning?

Reply:
That's true, and it makes this PR simpler, thank you!